### PR TITLE
Check hyperwallet client response content type

### PR DESCRIPTION
**Pull request by akreisman-epam**

### CODE REVIEW

File: src/utils/ApiClient.js [hunk 1]
```diff
@@ -125,6 +125,12 @@ export default class ApiClient {
      */
     wrapCallback(callback = () => null) {
         return (err, res) => {
+            if (res !== undefined && res.type !== "application/json" && res.type !== "application/jose+json") {
```
Comment (peter-joseph) [inline src/utils/ApiClient.js]: will the code easier to ready if we do the ff:
`if (res !== undefined && !(res.type === "application/json" || res.type === "application/jose+json")) {`

File: test/utils/ApiClient.spec.js [hunk 2]
```diff
@@ -489,5 +492,46 @@ describe("utils/ApiClient", () => {
             });
             callback(new Error(), rawRes);
         });
+
+        it("should call callback with 'body' and 'res' and application/jose+json Content-Type", (cb) => {
```
Comment (peter-joseph) [inline test/utils/ApiClient.spec.js]: Thanks for verifying the other content type from here, but please do too in Java

File: test/utils/ApiClient.spec.js [hunk 3]
```diff
@@ -489,5 +492,46 @@ describe("utils/ApiClient", () => {
...
+                err.should.be.deep.equal([{
+                    message: "Invalid Content-Type specified in Response Header",
+                }]);
+                body.should.be.equal("test");
+                rawRes.should.be.deep.equal(res);
+
+                cb();
+            });
+            callback(new Error(), rawRes);
+        });
```
Comment (peter-joseph) [inline test/utils/ApiClient.spec.js]: I rechecked the code based on requirements from the ticket, it seems like NodeJS SDK does not support encrypted request for payloads like `application/jose+json`
for example in ApiClient doGet:113 

it has this request
`request.accept("json")` -> `request.set('Accept', 'application/json')` 
if not encrypted request and `request.set('Accept', 'application/jose+json'` if encrypted respectively

File: test/utils/ApiClient.spec.js (hunk 2 above)
Comment (akreisman-epam) [reply test/utils/ApiClient.spec.js]: Ok

File: test/utils/ApiClient.spec.js (hunk 3 above)
Comment (akreisman-epam) [reply test/utils/ApiClient.spec.js]: Until PR #7 is not merged we can't define if request is encrypted or not so for now Accept header is always = 'application/json'

File: src/utils/ApiClient.js (hunk 1 above)
Comment (akreisman-epam) [reply src/utils/ApiClient.js]: Done

**Pull request merged**